### PR TITLE
(#21) - Add tests for `trustAll`

### PR DESCRIPTION
**Pull request by andreoss**
Per #22 
- Add more tests

### CODE REVIEW

File: src/test/java/com/artipie/http/client/jetty/JettyClientSlicesTest.java [hunk 1]
```diff
@@ -318,4 +316,67 @@ void shouldNotTimeoutIdleConnectionIfDisabled() throws Exception {
...
+        } finally {
+            client.stop();
+        }
+        Assertions.assertNotNull(caught);
+        MatcherAssert.assertThat(
+            caught,
+            Matchers.hasProperty(
+                "cause",
+                Matchers.isA(SSLException.class)
+            )
```
Comment (g4s8) [inline src/test/java/com/artipie/http/client/jetty/JettyClientSlicesTest.java:379]: You can use `Assertions.assertThrows` from Junit5 here:
```java
SSLException ex = Assertions.assertThrows(SSLException.class, () -> client.https(url).response(...));
```

File: src/test/java/com/artipie/http/client/jetty/JettyClientSlicesTest.java [hunk 2]
```diff
@@ -318,4 +316,67 @@ void shouldNotTimeoutIdleConnectionIfDisabled() throws Exception {
...
+        Exception caught = null;
+        try {
+            client.start();
+            MatcherAssert.assertThat(
+                client.https(url).response(
+                    new RequestLine(RqMethod.GET, "/").toString(),
+                    Headers.EMPTY,
+                    Flowable.empty()
+                ),
+                new RsHasStatus(RsStatus.OK)
```
Comment (g4s8) [inline src/test/java/com/artipie/http/client/jetty/JettyClientSlicesTest.java:365]: Checking status is confusing here: if we expect this code to fail with exception, why do we need this assertion?

**Review by g4s8 — CHANGES_REQUESTED** [review]
@andreoss thanks, please check my comments

File: src/test/java/com/artipie/http/client/jetty/JettyClientSlicesTest.java (hunk 2 above)
Comment (andreoss) [reply src/test/java/com/artipie/http/client/jetty/JettyClientSlicesTest.java]: @g4s8 Done.

File: src/test/java/com/artipie/http/client/jetty/JettyClientSlicesTest.java (hunk 1 above)
Comment (andreoss) [reply src/test/java/com/artipie/http/client/jetty/JettyClientSlicesTest.java]: @g4s8 Done.

**Review by g4s8 — APPROVED** [review]
@andreoss thanks

**Comment by kirill-h5i** [comment]
PR for #21

**Pull request merged**